PowerEditor/src/Parameters.h

```diff
@@ -897,6 +897,7 @@ struct NppGUI final
 	int _fileSwitcherExtWidth = 50;
 	bool _fileSwitcherWithoutPathColumn = true;
 	int _fileSwitcherPathWidth = 50;
+	bool _fileSwitcherDisableGroups = true;

```

Comment:
Why should it be defaulted to `true`?
If you delete your config.xml, Groups would be disabled.

`_fileSwitcherWithoutExtColumn` was *mistakenly* changed to `true` when `_fileSwitcherWithoutPathColumn` was added.